plugins/modules/dconf.py

```diff
@@ -26,6 +26,13 @@
     C(dconf), C(dbus-send), and C(dbus-run-session) binaries. Depending on
     distribution you are using, you may need to install additional packages to
     have these available.
+  - This module uses the C(gi.repository.GLib) Python library when available
+    for accurate comparison of values in C(dconf) to values specified in
+    Ansible code. C(gi.repository.GLib) is likely to be present on most systems
```

Comment:
Please also mention this library under `requirements:`.

Reply:
Done, please confirm.